runtime/modules/isp_vcu118.py

```diff
@@ -190,8 +190,15 @@ def send_command(com):
         gdb_log.close()
 
 
-def ap_thread(ap_tty, ap_log, runtime):
-    ap_serial = serial.Serial(ap_tty, 115200, timeout=3000000, bytesize=serial.EIGHTBITS,
+def ap_thread(ap_tty, ap_log, runtime, processor):
+    baud_rate = 0
+
+    if processor == "P1":
+        baud_rate = 57600
+    elif processor == "P2":
+        baud_rate = 115200
```

Comment:
Add another option for P3

Reply:
Since we don't have a way to test applications on P3 yet, I don't think it makes sense to support an untested option.